src/its_preselector/web_relay_preselector.py

```diff
@@ -30,12 +30,18 @@ def set_state(self, i):
             raise Exception("RF path " + key + " configuration does not exist.")
 
     def get_sensor_value(self, sensor_num):
-        sensor_num_string = str(sensor_num)
-        response = requests.get(self.base_url + '?sensor' + sensor_num_string)
+        sensor_num_string =  str(sensor_num)
+        response = requests.get(self.base_url)
         sensor_tag = 'sensor' + sensor_num_string
         root = ET.fromstring(response.text)
         sensor = root.find(sensor_tag)
-        return sensor.text
+        if sensor is None:
+            sensor_tag = 'oneWireSensor' + sensor_num_string
+            sensor = root.find(sensor_tag)
+        if sensor is None:
```

Comment:
Should this if statement be nested inside the above one, after `sensor = root.find(sensor_tag)` ?